src/dfind/dfind.c

```diff
@@ -302,6 +360,12 @@ int main (int argc, char** argv)
         {"quiet",     0, 0, 'q'},
         {"help",      0, 0, 'h'},
 
+	/* DAOS specific options */
+        { "pool"                , required_argument, 0, 'x' },
```

Comment:
Let's namespace the daos-specific options to avoid name collisions with other options / storage systems.  Something like "daos-pool" instead of just "pool"?

Reply:
@adammoody fixed in #339